js/bootstrap-datepicker.js

```diff
@@ -669,10 +669,11 @@
 			var calendarWidth = this.picker.outerWidth(),
 				calendarHeight = this.picker.outerHeight(),
 				visualPadding = 10,
-				windowWidth = $(this.o.container).width(),
-				windowHeight = $(this.o.container).height(),
-				scrollTop = $(this.o.container).scrollTop(),
-				appendOffset = $(this.o.container).offset();
+				container = $(this.o.container),
```

Comment:
I like to name jQuery variables with starting `$` too, but I don't see this tradition in this repo.